src/languages/en.js

```diff
@@ -29,26 +29,30 @@ export default {
     },
     attachmentPicker: {
         cameraPermissionRequired: 'Camera Permission Required',
-        expensifyDoesntHaveAccessToCamera: 'Expensify.cash does not have access to your camera, please enable the permission and try again.',
+        expensifyDoesntHaveAccessToCamera:
+            'Expensify.cash does not have access to your camera, please enable the permission and try again.',
         attachmentError: 'Attachment Error',
-        errorWhileSelectingAttachment: 'An error occurred while selecting an attachment, please try again',
+        errorWhileSelectingAttachment:
+            'An error occurred while selecting an attachment, please try again',
         errorDuringAttachmentSelection: 'Error during attachment selection',
         takePhoto: 'Take Photo',
         chooseFromGallery: 'Choose from Gallery',
         chooseDocument: 'Choose Document',
     },
     textInputFocusable: {
         noExtentionFoundForMimeType: 'No extension found for mime type',
-        problemGettingImageYouPasted: 'There was a problem getting the image you pasted',
+        problemGettingImageYouPasted:
+            'There was a problem getting the image you pasted',
     },
     baseUpdateAppModal: {
         updateApp: 'Update App',
-        updatePrompt: 'A new version of Expensify.cash is available.\nUpdate now or restart the app at a later time to download the latest changes.',
+        updatePrompt:
+            'A new version of Expensify.cash is available.\nUpdate now or restart the app at a later time to download the latest changes.',
     },
     iOUConfirmationList: {
         whoPaid: 'WHO PAID?',
         whoWasThere: 'WHO WAS THERE?',
-        whatsItFor: 'WHAT\'S IT FOR?',
+        whatsItFor: "WHAT'S IT FOR?",
```

Comment:
This doesn't respect the Airbnb Style Guide.
https://github.com/airbnb/javascript#strings--escaping